ios/RNCallKeep/RNCallKeep.m

```diff
@@ -133,8 +135,20 @@ + (void)initCallKitProvider {
     }
 }
 
++ (void)setup:(NSDictionary *)options {
+    RNCallKeep *callKeep = [RNCallKeep allocWithZone: nil];
+    [callKeep setup:options];
+}
+
 RCT_EXPORT_METHOD(setup:(NSDictionary *)options)
 {
+    if (_isSetup) {
+#ifdef DEBUG
+        NSLog(@"[RNCallKeep][setup] already setup");
+#endif
+        return;
+    }
+    
```

Comment:
If we've already called setup, we ignore subsequent calls to this method.

I'm not sure if this is something we'd want. Is it expected to be able to change the setup during the life cycle?

Reply:
I wouldnt say so